scripts/secnetperf-helpers.psm1

```diff
@@ -521,11 +533,14 @@ function Invoke-Secnetperf {
         Write-Host "==============================`nRUN $($try+1):"
         "> secnetperf $clientArgs" | Add-Content $clientOut
         try {
-            $process = Start-LocalTest $clientPath $clientArgs $artifactDir
+            $process = Start-LocalTest "$sudo$clientPath" $clientArgs $artifactDir
             $rawOutput = Wait-LocalTest $process $artifactDir ($io -eq "wsk") 30000
             Write-Host $rawOutput
             $values[$tcp] += Get-TestOutput $rawOutput $metric
             if ($extraOutput) {
+                if ($sudo -ne "") {
+                    sudo chown $UserName $extraOutput
```

Comment:
Why is this necessary?

Reply:
The extraOutput (latency.txt) is owned by root when running secnetperf with sudo. deleting the file from normal user get failed. Existence of latency.txt make next run fail